pkg/kubelet/kubelet_test.go

```diff
@@ -311,7 +311,7 @@ func newTestKubeletWithImageList(
 	kubelet.resyncInterval = 10 * time.Second
 	kubelet.workQueue = queue.NewBasicWorkQueue(fakeClock)
 	// Relist period does not affect the tests.
-	kubelet.pleg = pleg.NewGenericPLEG(fakeRuntime, 100, time.Hour, kubelet.podCache, clock.RealClock{})
+	kubelet.pleg = pleg.NewGenericPLEG(fakeRuntime, make(chan *pleg.PodLifecycleEvent, plegChannelCapacity), genericPlegRelistPeriod, genericPlegRelistThreshold, kubelet.podCache, clock.RealClock{})
```

Comment:
why we are changing the super long test relist period (1 hr) to the real period?

Reply:
cc @swghosh

Reply:
I just reverted those values to use relistPeriod: 100s, relistThreshold: 1hr (to retain the old values as is) and apparently it still works fine.